src/NServiceBus.Metrics.PerformanceCounters/PerformanceCountersFeature.cs

```diff
@@ -0,0 +1,76 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using NServiceBus;
+using NServiceBus.Features;
+
+class PerformanceCountersFeature : Feature
+{
+    public PerformanceCountersFeature()
+    {
+        Defaults(s =>
+        {
+            options = s.EnableMetrics();
+            s.SetDefault(UpdateIntervalKey, TimeSpan.FromSeconds(2));
+        });
+    }
+
+    protected override void Setup(FeatureConfigurationContext context)
+    {
+        var logicalAddress = context.Settings.LogicalAddress();
+
+        var legacyInstanceNameMap = new Dictionary<string, CounterInstanceName?>
```

Comment:
@danielmarbach My mistake about JSON.NET. For some reason, I thought it was in the mapper, not the updater. 

I'm confused about the first part though. I thought @mikeminutillo's suggestion was to rename the counters in `NServiceBus.Metrics` so that they match what we have always used. In which case, we could get rid of all the code that maps the old news to the new ones.